base/io.jl

```diff
@@ -647,11 +647,11 @@ Commit all currently buffered writes to the given stream.
 flush(io::IO) = nothing
 
 """
-    skipchars(io::IO, predicate; linecomment=nothing)
+    skipchars(io::IO, predicate; linecomment::Char)
 
-Skip forward in `io` until `predicate` returns `false`. If `linecomment`
-is defined, all characters after the `linecomment` character are ignored
-until the next line.
+Advance the stream `io` until before the first character for which `predicate`
```

Comment:
"until before the first character for which `predicate` returns false" seems potentially confusing: Will the predicate return false for the next-read character, or true for the next-read character and then false for the character thereafter? Perhaps "Advance the stream `io` to the first character for which..." would be clearer?